src/extensions/changelog.ts

```diff
@@ -5,9 +5,10 @@ import * as semver from "semver";
 import TerminalRenderer = require("marked-terminal");
 import Table = require("cli-table");
 
-import { listExtensionVersions, parseRef } from "./extensionsApi";
+import { listExtensionVersions } from "./extensionsApi";
 import { readFile } from "./localHelper";
 import { logger } from "../logger";
+import * as ref from "./ref";
```

Comment:
nit: I do like the usage of `refHelper` more than `ref` because it is easier to find all references to this file. Perhaps the file itself should also be named `refHelper` for the same reason, and also for consistency with our existing pattern.

Reply:
Chatted elsewhere - renaming this to refs to avoid the xHelper pattern that we've fallen into and also to avoid collision with all of the other uses of ref throughout the codebase